spec/factories/application_merits_task/involved_children.rb

```diff
@@ -0,0 +1,9 @@
+module ApplicationMeritsTask
```

Comment:
Sorry, boring comment. Does this need to be pluralised along with the parent dir for consistency? I know we do this with the CFE Results factory so just checking.